<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atleto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/main/icons/runner32.xpm</normaloff>:/main/icons/runner32.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainContent">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>460</height>
       </size>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="CalendarWidget" name="calendarPage"/>
      <widget class="ReportWidget" name="reportPage">
       <property name="styleSheet">
        <string/>
       </property>
      </widget>
      <widget class="SearchRunWidget" name="searchRunPage"/>
      <widget class="RacesWidget" name="racesPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_TCX"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAthlete"/>
    <addaction name="actionShoes"/>
    <addaction name="separator"/>
    <addaction name="actionCalendar"/>
    <addaction name="actionReport_Generator"/>
    <addaction name="actionSearch_Run"/>
    <addaction name="actionRaces"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Atleto"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRun_Calculator"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalendar"/>
   <addaction name="actionReport_Generator"/>
   <addaction name="actionSearch_Run"/>
   <addaction name="actionRaces"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Atleto">
   <property name="text">
    <string>About Atleto</string>
   </property>
  </action>
  <action name="actionReport_Generator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/main/icons/report.png</normaloff>:/main/icons/report.png</iconset>
   </property>
   <property name="text">
    <string>Report Generator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAthlete">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/main/icons/athlete.png</normaloff>:/main/icons/athlete.png</iconset>
   </property>
   <property name="text">
    <string>Athlete...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSearch_Run">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/main/icons/find.png</normaloff>:/main/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Search Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRun_Calculator">
   <property name="text">
    <string>Run Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShoes">
   <property name="text">
    <string>Shoes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/main/icons/cal.png</normaloff>:/main/icons/cal.png</iconset>
   </property>
   <property name="text">
    <string>Calendar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/main/icons/trophy.xpm</normaloff>:/main/icons/trophy.xpm</iconset>
   </property>
   <property name="text">
    <string>Races</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>plot</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionImport_TCX">
   <property name="text">
    <string>Import TCX...</string>
   </property>
  </action>
  <action name="actionPlotTest">
   <property name="text">
    <string>PlotTest</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ReportWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/reportwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchRunWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/searchrunwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/calendarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RacesWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/raceswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
